### PR TITLE
[Add] Support HOTA evaluation for MOTChallenge.

### PR DESCRIPTION
**Pull request by Seerkfang**

### CODE REVIEW

File: mmtrack/datasets/mot_challenge_dataset.py [hunk 1]
```diff
@@ -240,6 +245,73 @@ def format_bbox_results(self, results, infos, resfile):
...
+            GT_FOLDER=self.img_prefix,
+            # Trackers location
+            TRACKERS_FOLDER=tracker_folder,
+            # Where to save eval results
+            # (if None, same as TRACKERS_FOLDER)
+            OUTPUT_FOLDER=None,
+            # Use 'track' as the default tracker
+            TRACKERS_TO_EVAL=['track'],
+            # Option values: ['pedestrian']
+            CLASSES_TO_EVAL=['pedestrian'],
```
Comment (OceanPang) [inline mmtrack/datasets/mot_challenge_dataset.py:268]: We have `self.classes`, is `CLASSES_TO_EVAL` related to this item?

File: mmtrack/datasets/mot_challenge_dataset.py [hunk 2]
```diff
@@ -240,6 +245,73 @@ def format_bbox_results(self, results, infos, resfile):
...
+            # Option Values: 'MOT17', 'MOT16', 'MOT20', 'MOT15'
+            BENCHMARK=self.img_prefix.rsplit('/', 2)[-2],
```
Comment (OceanPang) [inline mmtrack/datasets/mot_challenge_dataset.py:270]: This is risky, write a function to convert lower-case names, and raise errors if there are no specific benchmarks in their names.

File: mmtrack/datasets/mot_challenge_dataset.py [hunk 3]
```diff
@@ -311,17 +383,60 @@ def evaluate(self,
...
-            print(str_summary)
+
+            if trackeval is None:
+                raise ImportError(
+                    'Please run'
+                    'pip install git+https://github.com/JonathonLuiten/TrackEval.git'  # noqa
+                    'to manually install trackeval')
+
+            seqmap = osp.join(self.img_prefix, 'videoseq.txt')
+            if not osp.exists(seqmap):
```
Comment (OceanPang) [inline mmtrack/datasets/mot_challenge_dataset.py:394]: Add document instructions to indicate dataset preparations.

File: mmtrack/datasets/mot_challenge_dataset.py [hunk 4]
```diff
@@ -311,17 +390,62 @@ def evaluate(self,
...
-            print(str_summary)
+
+            if trackeval is None:
+                raise ImportError(
+                    'Please run'
+                    'pip install git+https://github.com/JonathonLuiten/TrackEval.git'  # noqa
+                    'to manually install trackeval')
+
+            # get video name sequence for HOTA evaluation
+            seqmap = osp.join(self.img_prefix, 'videoseq.txt')
```
Comment (GT9505) [inline mmtrack/datasets/mot_challenge_dataset.py:401]: Can we online generate this txt ?

File: mmtrack/datasets/mot_challenge_dataset.py [hunk 5]
```diff
@@ -240,6 +245,80 @@ def format_bbox_results(self, results, infos, resfile):
...
+        BENCHMARKS = ['MOT15', 'MOT16', 'MOT17', 'MOT20']
+        for benchamrk in BENCHMARKS:
+            if benchamrk in self.img_prefix.upper():
+                return benchamrk
+
+    def get_dataset_cfg_for_hota(self, tracker_folder, seqmap):
+        """Get default configs for trackeval.datasets.MotChallenge2DBox.
+
+        Args:
+            tracker_folder (str): the name of the tracker folder
```
Comment (GT9505) [inline mmtrack/datasets/mot_challenge_dataset.py:259]: compelete docstring

**Comment by GT9505** [comment]
change CI

File: docs/zh_cn/install.md [hunk 6]
```diff
@@ -51,7 +51,16 @@
 
     如果不是安装预构建的包，而是从源码中构建 PyTorch，则可以使用更多的 CUDA 版本，例如 CUDA 9.0。
 
-3. 安装 mmcv-full，我们推荐您安装以下预构建包：
+3. 为VOT数据集测试评估安装额外库（可选）
```
Comment (GT9505) [inline docs/zh_cn/install.md:54]: typically, we need add a blank space between Chinese and English

**Review by GT9505 — APPROVED** [review]

**Pull request merged**